sphinx_ncs_theme/static/css/nordic.css

```diff
@@ -596,3 +596,31 @@ div.figure {
 .contents.local ul p {
   margin-bottom: 0px;
 }
+
+.rst-content table.docutils td p {
+  margin-bottom: 0.6rem !important;
+}
+
+.rst-content table.docutils td p:last-child {
+  font-size: .9rem;
+  margin-bottom: 0 !important;
+}
+
+a.reference.internal[href*="#config-"] {
+  color: #404040 !important;
```

Comment:
This should be clubbed with the block starting at line 468

Reply:
Done.